assets/src/js/Components/archivePicker/archivePickerSidebar.js

```diff
@@ -0,0 +1,22 @@
+import { SingleSidebar } from './SingleSidebar';
+import { MultiSidebar } from './MultiSidebar';
+
+export const archivePickerSidebar = ( archivePicker ) => ( imagePicker ) => {
```

Comment:
I initially added the `archivePicker` argument as a quick fix after moving this render prop function outside of it, with the intention of replacing it later by passing on only certain props. But actually this way might be more clear about the intention of this function, i.e. it should provide the `imagePicker` sidebar for the `ArchivePicker` component, and the signature is also recognizable.